PHASE_2_QUICK_REFERENCE.md

```
@@ -0,0 +1,164 @@
+## Testing Imports
+
+```bash
+# All three should succeed:
+poetry run python -c "from omega_kg.config import SUPPORTED_NODE_TYPES; print('✓')"
+poetry run python -c "from omega_kg.vector_store import VectorStore; print('✓')"
+poetry run python -c "from omega_kg.workers.embedding_worker import EmbeddingWorker; print('✓')"
+```
+
+## File Locations
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)